examples/ptmatchadapter-fril/src/main/resources/application.properties

```diff
@@ -0,0 +1,26 @@
+# BANNER
+#banner.charset=UTF-8 # Banner file encoding.
+#banner.location=classpath:banner.txt # Banner file location.
+
+# APPLICATION SETTINGS (SpringApplication)
+#spring.main.banner-mode=console # Mode used to display the banner when the application runs.
+
+# build fails if mainClass not set in build.gradle
+# camel-spring-boot wants it set here  when main app extends FatJarRouter
+#  - see http://camel.apache.org/spring-boot.html
+spring.main.sources=org.mitre.ptmatchadapter.PtmatchAdapter
+
+# Spring Boot enables a JMX listener by default
+camel.springboot.jmxEnabled = false
+
+# URL to the FHIR server from which to request messages. 
+# Include scheme, host name + port + root path
+src.fhir.server.base = http://localhost:8881
+
+ptmatchadapter.workDir = /home/michael/Documents/fril-runs
```

Comment:
The FHIR server base url and path to the work directory are in a configuration file so a user can set them to anything they want.

I can look into using a relative path for the work directory so that a default value under adapter’s installation folder is assumed.

From: Andy Gregorowicz [mailto:notifications@github.com]
Sent: Wednesday, March 2, 2016 4:08 PM
To: mitre/ptmatchadapter <ptmatchadapter@noreply.github.com>
Cc: Los, Michael E. <mel@mitre.org>
Subject: Re: [ptmatchadapter] Initial commit of patient match adapter classes w/ no-op example (#1)


In examples/ptmatchadapter-fril/src/main/resources/application.properties<https://github.com/mitre/ptmatchadapter/pull/1#discussion_r54791388>:

> +# APPLICATION SETTINGS (SpringApplication)

> +#spring.main.banner-mode=console # Mode used to display the banner when the application runs.

> +

> +# build fails if mainClass not set in build.gradle

> +# camel-spring-boot wants it set here  when main app extends FatJarRouter

> +#  - see http://camel.apache.org/spring-boot.html

> +spring.main.sources=org.mitre.ptmatchadapter.PtmatchAdapter

> +

> +# Spring Boot enables a JMX listener by default

> +camel.springboot.jmxEnabled = false

> +

> +# URL to the FHIR server from which to request messages.

> +# Include scheme, host name + port + root path

> +src.fhir.server.base = http://localhost:8881

> +

> +ptmatchadapter.workDir = /home/michael/Documents/fril-runs

This will only work on your system. Is there any way to make it more generic?

—
Reply to this email directly or view it on GitHub<https://github.com/mitre/ptmatchadapter/pull/1/files#r54791388>.